2.2.0

gabrielcsapo opened the pull request:
- add feature for ignoring environment variables
- Use load-jsonic-sync to load json with comments and report position and filename in error messages

grawk commented:
I think this should be a "minor" (new functionality). Also note that this repo has been using the "releases" feature:
https://github.com/krakenjs/confit/releases

Commit:
2.2.0
- add feature for ignoring environment variables
- Use load-jsonic-sync to load json with comments and report position and filename in error messages

gabrielcsapo commented:
@grawk updated

grawk reviewed (approved):
Looks good to me!
